src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Should `import static org.junit.jupiter.api.Assertions.assertEquals;` be above `import org.junit.jupiter.api.Test;`?